source/guide_soju.rst

```diff
@@ -0,0 +1,139 @@
+.. highlight:: console
+
+.. author:: apoxa <https://github.com/apoxa>
+
+.. tag:: lang-go
+.. tag:: irc
+.. tag:: bouncer
+
+.. spelling::
+    Soju
+    soju
+    upstreams
+
+##########
+Soju
+##########
+
+.. tag_list::
+
+soju_ is a user-friendly IRC bouncer which supports multiple users, multiple clients for a single user with backlog synchronization and multiple upstreams via a single IRC connection to the bouncer.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Firewall Ports <basics-ports>`
+  * :manual:`supervisord <daemons-supervisord>`
+
+License
+=======
+
+All relevant legal information can be found here:
+
+  * https://git.sr.ht/~emersion/soju/tree/master/item/LICENSE
+
+Prerequisites
+=============
+
+.. include:: includes/open-port.rst
+
+Installation
+============
+
+Clone the source into your home directory and enter the directory:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ git clone https://git.sr.ht/~emersion/soju
+ [isabell@stardust ~]$ cd ~/soju
+ [isabell@stardust soju]$
+
+Compile and install soju to your home directory:
+
+.. code-block:: console
+
+ [isabell@stardust soju]$ make -W doc/soju.1
+ [...]
+ [isabell@stardust soju]$ PREFIX="/.local" DESTDIR="$HOME" make install
+ [...]
+ [isabell@stardust soju]$
+
+Configuration
+=============
+
+Configure soju
+--------------
+
+Save the following as your soju configuration in ``~/etc/soju/config``.
+
+.. code-block:: ini
+
+ listen ircs://0.0.0.0:40132
+ tls /home/isabell/etc/certificates/isabell.uber.space.crt /home/isabell/etc/certificates/isabell.uber.space.key
```

Comment:
is it possible to use the tilde ~ expansion for /home/isabell here?

Reply:
I tested this and unfortunately it's neither possible to use ~ nor $HOME for the path here.